_opensearch/install/compatibility.md

```diff
@@ -8,12 +8,12 @@ nav_order: 2
 # Operating system and JVM compatibility
 
 - We recommend installing OpenSearch on RHEL- or Debian-based Linux distributions that use [systemd](https://en.wikipedia.org/wiki/Systemd), such as CentOS, Amazon Linux 2, and Ubuntu (LTS). OpenSearch should work on many Linux distributions, but we only test a handful.
-- The OpenSearch tarball ships with a compatible version of Java in the `jdk` directory. To find its version, run `./jdk/bin/java -version`. For example, the OpenSearch 1.0.0 tarball ships with Java 15 (non-LTS).
+- The OpenSearch tarball ships with a compatible version of Java in the `jdk` directory. To find its version, run `./jdk/bin/java -version`. For example, the OpenSearch 1.0.0 tarball ships with Java 15.0.1+9 (non-LTS), while OpenSearch 1.3.0 includes Java 11.0.14.1+1 (LTS).
+- OpenSearch 1.0 to 1.2.x is built and tested with Java 15, while OpenSearch 1.3.0 is built and tested with Java 8, 11 and 14.
```

Comment:
Still new here. Is there a possibility that we'll still make patches for 1.2.x? If not, should we switch from "1.2.x" to "1.2.4"?